recipes/golang.rb

```diff
@@ -0,0 +1,5 @@
+# Use golang recipe to install bosun
+default['go']['packages']           = ['bosun.org/cmd/bosun']
+default['bosun']['install_path']    = "#{node['go']['gobin']}/bosun"
```

Comment:
Right catch, thanks!. It should be as `node.default`. I'll update the pull request shortly.